src/App.js

```diff
@@ -31,7 +31,7 @@ class App extends Component {
       const movies = data.Search || []
       const color = data.Error ? 'red' : 'black'
       this.setState({ status, movies, color })
-    })
+    }).catch((error) => this.setState({ status: error.message || 'Something went wrong...', color: 'red'  }))
```

Comment:
Just move this to its own line. The convention is:

```
axios.get()
  .then()
  .then()
  .catch()
```

Reply:
thanks. done.